### PR TITLE
add feature windows to support windows

### PR DESCRIPTION
**Pull request by teamplayer3**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qr2term"
-version = "0.3.2"
+version = "0.3.3"
```
Comment (timvisee) [inline Cargo.toml:3]: Please do not increase the version number here. That'll be done separately.

File: Cargo.toml [hunk 2]
```diff
@@ -9,14 +9,13 @@ repository = "https://github.com/timvisee/qr2term-rs"
...
-    "rendering",
-    "visualization",
-]
+categories = ["algorithms", "rendering", "visualization"]
 edition = "2021"
 rust-version = "1.67.1"
 
+[features]
+windows = ["crossterm/windows"]
+
```
Comment (timvisee) [inline Cargo.toml:18]: Would it be better to enable this feature by default on Windows?

**Review by timvisee — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (teamplayer3) [reply Cargo.toml:18]: Yes, great idea.

Comment (timvisee) [reply Cargo.toml:18]: Yes, I had the option in your second link in mind.

It's not nice, but the best we can do for now.

Comment (timvisee) [reply Cargo.toml:18]: On second thought, it seems we can enable it by default in all cases, so also on non-Windows platforms. It seems that crossterm doesn't include any dependencies when the feature is set if compiling on non-Windows.

Comment (teamplayer3) [reply Cargo.toml:18]: Great idea. I will look at it later this day.

**Comment by timvisee** [comment]
@teamplayer3 Thanks for the PR!

I've just released version 0.3.3 which includes this change: https://crates.io/crates/qr2term

**Pull request merged**